sdk/ml/azure-ai-ml/azure/ai/ml/entities/_monitoring/input_data.py

```diff
@@ -2,28 +2,26 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from azure.ai.ml.entities._mixins import RestTranslatableMixin
 from azure.ai.ml._restclient.v2023_04_01_preview.models import MonitoringInputData as RestMonitoringInputData
-from azure.ai.ml._utils.utils import camel_to_snake, snake_to_camel
 from azure.ai.ml._utils._experimental import experimental
-
+from azure.ai.ml._utils.utils import camel_to_snake, snake_to_camel
 from azure.ai.ml.constants._monitoring import MonitorDatasetContext
 from azure.ai.ml.entities._inputs_outputs import Input
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
 
 
 @experimental
 class MonitorInputData(RestTranslatableMixin):
-    """Monitor input data
+    """Monitor input data.
 
     :param input_dataset: Input data used by the monitor
-    :type input_dataset: ~azure.ai.ml.entities.Input
-    :param dataset_context: The context of the input dataset. Possible values
-        include: model_inputs, model_outputs, training, test, validation,
-        ground_truth
-    :type dataset_context: str or ~azure.ai.ml.constants.MonitorDatasetContext
-    :param target_column_name: The target column in the given input dataset to leverage
+    :type input_dataset: ~azure.ai.ml.Input
+    :param dataset_context: The context of the input dataset. Accepted values are "model_inputs",
+        "model_outputs", "training", "test", "validation", and "ground_truth".
+    :type dataset_context: Union[str, ~azure.ai.ml.constants.MonitorDatasetContext]
+    :param target_column_name: The target column in the given input dataset to leverage.
```

Comment:
"The target column in the given input dataset to leverage."

Leverage in what way?

Reply:
Honestly, idk. This was the existing description, and I assumed this was language that would be familiar to anyone doing dataset monitoring, but outside of the references to our SDK documentation, I don't see this language anywhere. I'm going to remove it.